forte/processors/base/batch_processor.py

```diff
@@ -288,24 +292,61 @@ def default_configs(cls) -> Dict[str, Any]:
             "feature_scheme": None,
             "batch_size": None,
             "model": None,
-            "batcher": cls.define_batcher().default_configs()
+            "batcher": cls.define_batcher().default_configs(),
+            "do_eval": False
         })
         return super_config
 
     def initialize(self, resources: Resources, configs: Optional[Config]):
         if configs is not None:
+            configs = self._parse_configs(configs)
             batcher_config = {"scope": configs.scope, "feature_scheme": {}}
             for tag, scheme in configs.feature_scheme.items():
                 if scheme["type"] == TrainPreprocessor.DATA_INPUT:
                     batcher_config["feature_scheme"][tag] = scheme
             batcher_config["batch_size"] = configs.batch_size
             configs.batcher = batcher_config
+            self.do_eval = configs.do_eval
 
         super().initialize(resources, configs)
 
     def load(self, model):
         self.model = model
 
+    def _parse_configs(self, configs):
+        parsed_configs = self.default_configs()
+        parsed_configs["batch_size"] = configs.batch_size
+        parsed_configs['scope'] = get_class(configs.scope)
+        parsed_configs['do_eval'] = configs.do_eval
+        parsed_configs['feature_scheme'] = {}
+        for tag, scheme in configs.feature_scheme.items():
+            parsed_configs["feature_scheme"][tag] = {}
+            if scheme["type"] == "data_input":
+                parsed_configs["feature_scheme"][tag]["type"] = \
+                    TrainPreprocessor.DATA_INPUT
+            elif scheme["type"] == "data_output":
+                parsed_configs["feature_scheme"][tag]["type"] = \
+                    TrainPreprocessor.DATA_OUTPUT
+
+            extractor = get_class(
+                scheme["extractor"]["class_name"])()
+            extractor.initialize(
+                config=scheme["extractor"]["config"])
+            if "vocab_path" in scheme["extractor"]:
+                vocab_file = open(scheme["extractor"]["vocab_path"], 'rb')
+                extractor.vocab = pickle.load(vocab_file)
+                vocab_file.close()
+            parsed_configs["feature_scheme"][tag]["extractor"] = \
+                extractor
+
+            if "converter" not in scheme:
+                parsed_configs["feature_scheme"][tag]["converter"] = \
```

Comment:
So we are storing the objects inside the `Config`, can we just store them in some other dict? We probably don't want to demo that configs can have objects/instances.

Fore example, we just store something named `self.feature_scheme`?